source/adminguide/locale/pot/storage.pot

```diff
@@ -1011,7 +1011,7 @@ msgstr ""
 
 #: ../../storage.rst:685
 # bc81c587ad8b4032b27d61690390e258
-msgid "With each snapshot schedule, users can also specify the number of scheduled snapshots to be retained. Older snapshots that exceed the retention limit are automatically deleted. This user-defined limit must be equal to or lower than the global limit set by the CloudStack administrator. See `“Globally Configured Limits” <usage.html#globally-configured-limits>`_. The limit applies only to those snapshots that are taken as part of an automatic recurring snapshot policy. Additional manual snapshots can be created and retained."
+msgid "With each snapshot schedule, users can also specify the number of recurring snapshots to be retained. Older snapshots that exceed the retention limit are automatically deleted. This user-defined limit must be equal to or lower than the global limit set by the CloudStack administrator. See `“Globally Configured Limits” <usage.html#globally-configured-limits>`_. The limit applies only to those snapshots that are taken as part of an automatic recurring snapshot policy. Additional manual snapshots can be created and retained."
```

Comment:
doesn't it makes more sense to be like: 

With each recurring snapshot, users can also specify the number of snapshots to be retained.

Reply:
Yes, how about "With each reccurring snapshot schedule, users can also specify the number of recurring snapshots to be retained."?

This is similar to the message shown in the Recurring Snapshots form.
<img width="620" alt="Screenshot 2024-10-29 at 2 16 37 PM" src="https://github.com/user-attachments/assets/07520f73-90f5-4cb1-9e37-92c5259b38d1">